src/libstd/panic.rs

```diff
@@ -231,6 +231,34 @@ impl<T: ?Sized> RefUnwindSafe for Mutex<T> {}
 #[stable(feature = "unwind_safe_lock_refs", since = "1.12.0")]
 impl<T: ?Sized> RefUnwindSafe for RwLock<T> {}
 
+#[stable(feature = "unwind_safe_atomic_refs", since = "1.14.0")]
+impl RefUnwindSafe for atomic::AtomicIsize {}
+#[stable(feature = "unwind_safe_atomic_refs", since = "1.14.0")]
```

Comment:
It doesn't make much sense for this to be marked stable when `AtomicI8` is still unstable. I think for the unstable types this should be `#[unstable(feature = "integer_atomics", issue = "32976")]` like the others impls for these types.